ruby-v2/README.md

```diff
@@ -5,13 +5,13 @@ Algolia's API Client ruby was created by, and is maintained by [Algolia](https:/
 ## Get started locally
 
 > **Requires:**
-- **[Ruby > 2.2](https://www.ruby-lang.org)**
+- **[Ruby > 1.8.7](https://www.ruby-lang.org)**
```

Comment:
Why did we change this? We only support 2.2 and up right? 🤔 

Edit: I think you can revert the changes to this file, right? It doesn't seem to be correct 🙂